frame/staking/src/pallet/mod.rs

```diff
@@ -1374,6 +1376,7 @@ pub mod pallet {
 		pub fn force_no_eras(origin: OriginFor<T>) -> DispatchResult {
 			ensure_root(origin)?;
 			ForceEra::<T>::put(Forcing::ForceNone);
+			Self::deposit_event(Event::<T>::ForceEra { mode: Forcing::ForceNone });
```

Comment:
probably set in other places as well. 

A setter function for this would be good.